cuqipy_fenics/geometry.py

```diff
@@ -245,8 +257,10 @@ def par2field(self, p):
         Ns = p.shape[-1]
         field_list = np.empty((self.geometry.par_dim,Ns))
 
+        nu = self.nu
+        d = self.physical_dim
         for idx in range(Ns):
-            field_list[:,idx] = self.eig_vec@( np.sqrt(self.eig_val)*p[...,idx] )
+            field_list[:,idx] = self.eig_vec@( self.eig_val**((nu+d/2)/2)*p[...,idx] )
```

Comment:
Could ref where this eq. is found. Perhaps in docstring?

Reply:
@nabriis I added a particular reference.
@babakmaboudi, can you take a quick look at the reference I added, section 4.3, in particular last equation in the section. Thanks!